### PR TITLE
Added the .ipynb file and the .py file for Data Skew Augmentation

### LINKED ISSUE #4: Implement skew for dataset augmentation
* Currently the training dataset is augmented from a single pair of cloudy and cloud-free images
* The augmentation is performed only by rotation of 0, 90, 180 and -90 degrees which gives us 4 pairs, say 1,2,3 and 4
* These four pairs are then stacked iteratively 10 times as: ```[1,2,3,4,1,2,3,4,1,2]```

- [ ] Implement skew operations on the cloudy and cloud-free images, and then perform the rotation operations. You will now have 4 pairs of original *(unskewed)* images, say, [1, 2, 3, 4] and 4 pairs of the skewed images, say [1', 2', 3', 4']

- [ ] Prepare two new training datasets by stacking 10 times as ```[1,2,3,4,1',2',3',4',1,1']```

### PR DESCRIPTION
**Pull request by Preetesh21**
Fixes #4 
This PR contains the code for skewing the images and stacking them then using Keras API. I have attached the Notebook and Python File for reference and here is the Google Drive [Link](https://drive.google.com/file/d/1ghi_eFnIvyNMesgb8eMvkMHNdkBcnKp5/view?usp=sharing) for the Stack Data train and test.

### CODE REVIEW

**Comment by Chintan2108** [comment]
Hi @Preetesh21, good work on the code.
Here are a few guidelines to follow when making a PR:

- [ ] Always start your PR description with "Fixes #issue_number". I've edited your comment for this one, please refer to it. This not only links the issue to this PR, but also closes the issue when this PR is merged.
- [ ] Briefly mention the purpose of the PR, along with the tools/libraries you have used. It would be great if you could be version specific.
- [ ] Briefly mention what logic you used to implement the changes/upgrades.
- [ ] Provide in-code review comments on GitHub to highlight specific LOC if deemed necessary.
- [ ] Please provide snapshots if deemed necessary.
- [ ] Update readme if required.

You can make the following changes and make new commits under the same PR:
* Either update the sections 3.3 and 3.4 of the existing notebook with your code OR upload a new notebook OR upload a new python file. NOTE: Please do not upload the python file as generated from your Jupyter notebook console. 
* Name your new dataset as ```train_data_8_skew_10copy.npy```
* Upload it in the same drive folder (you can request edit access)
* Update readme
* Provide snapshots of the skewed images in your PR description

**Comment by Preetesh21** [comment]
Sir, here are the screenshots
TRAIN DATA
train
TEST DATA
test

I have removed the Python file and committed the changes. Could you kindly provide the edit access to the drive folder so that I can upload the skew dataset?

**Comment by Chintan2108** [comment]
@Preetesh21 Thanks for uploading the snapshots.
* I would like to know what skew value are you using?
* It seems you're applying a horizontal skew on the train data and a vertical skew on the test data. We want to apply the same skew to both train and test data, so that the model can establish a mapping function - the purpose of skewing is augmenting more data for the model to learn from, from the original single pair of scenes.
* Also, there seems to be loss of data on the edges in the skewed images. I think we can overcome this issue based on the skew value you are using. We don't want a very high skew - only a slight skew would do - so much so that at (i,j) location in skewed and unskewed images are different.
* Regarding drive access, please provide me with your gmail, on the gitter channel.
* Readme updation looks good.

**Comment by Preetesh21** [comment]
Sir, I am using a skew value of 0.3. So should I decrease it?

**Comment by Preetesh21** [comment]
I have updated the skew value(you were correct, I reduced the value) which is giving better results and also added the drive link in ReadMe.
Train 
train
Test
test

**Comment by Chintan2108** [comment]
@Preetesh21 The changes look good!

**Review by Chintan2108 — APPROVED** [review]

**Comment by Chintan2108** [comment]
give @Preetesh21 25 points
